### PR TITLE
DEPLOY-497 Update to keycloak stable chart

### PR DESCRIPTION
**Pull request by nicdoye**
Keycloak chart now includes `extraArgs`, `extraVolumes` and `extraVolumeMounts` to set realms, which removes the need for the InitContainer workaround.

`serviceName` was wrong before.

`README.md` now passes markdownlint checks.

Clarified section on using ACM SSL/TLS certs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,131 +56,180 @@ This environment variable will be used in the deployment steps.
...
+  --set ingressHostName=$ELBADDRESS \
+  --namespace $DESIREDNAMESPACE
 ```
 
 ## Customizing the Realm
 
 ### Customizing the Realm During Deployment
 
-1. You will need a realm file. A [sample realm](./alfresco-realm.json) file is provided.
+1. You will need a realm json, similar to `alfresco-realm.json`
```
Comment (kconkas) [inline README.md]: Just a minor note: why don't we want to point users directly to the sample realm file?

File: README.md [hunk 2]
```diff
@@ -22,18 +22,29 @@ These instructions illustrate deployment to a Kubernetes cluster on AWS.
 
 Please check the Anaxes Shipyard documentation on [running a cluster](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md).
 
-If you are deploying the Identity Service into a cluster with other Alfresco components such as Content Services and Process Services, a VPC and cluster with 5 nodes is recommended. Each node should be a m4.xlarge EC2 instance.
+Note the resource requirements:
+
+* Minikube: At least 2 gigs of memory, i.e.:
```
Comment (kconkas) [inline README.md]: Minor: as we're systematically stripping minikube references from other documents, maybe we should do it here too. I appreciate this is outside the scope of this ticket.

File: helm/alfresco-identity-service/values.yaml [hunk 3]
```diff
@@ -7,14 +7,23 @@ ingressHostName: "your ingress dns name"
...
     password: admin
+    extraVolumes: |
+      - name: realm-secret
+        secret:
+          secretName: realm-secret    
+    extraVolumeMounts: |
+      - name: realm-secret
+        mountPath: "/realm/"
+        readOnly: true
+    extraArgs: "-Dkeycloak.import=/realm/alfresco-realm.json"
```
Comment (kconkas) [inline helm/alfresco-identity-service/values.yaml]: 👍 for this approach to importing the realm

Comment (nicdoye) [reply helm/alfresco-identity-service/values.yaml]: It's a new feature added by Ryan Dawson to the official chart! 👍

Comment (svidrascu) [reply helm/alfresco-identity-service/values.yaml]: +1
